subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/accessors/ProjectSchemaProvider.kt

```diff
@@ -79,3 +79,14 @@ data class ProjectSchemaEntry<out T>(
     fun <U> map(f: (T) -> U) =
         ProjectSchemaEntry(f(target), name, f(type))
 }
+
+data class ConfigurationEntry<T>(
+    val target: T,
+    val dependencyDeclarationAlternatives: List<String> = listOf()
+) : Serializable {
```

Comment:
❓ Why `Serializable`?

Reply:
I just copied blindly the pattern of the other classes in this file, assuming this would be serialized at some point :)